### PR TITLE
Transfers - Initiate connection

### PR DESCRIPTION
**Pull request by elliotritchie**

### CODE REVIEW

File: src/SFA.DAS.EmployerApprenticeshipsService.Application/Queries/GetSentTransferConnectionInvitationQuery/SentTransferConnectionInvitationViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿using System.ComponentModel.DataAnnotations;
+using SFA.DAS.EAS.Domain.Models.TransferConnection;
+
+namespace SFA.DAS.EAS.Application.Queries.GetSentTransferConnectionInvitationQuery
+{
+    public class SentTransferConnectionInvitationViewModel
+    {
+        [Required]
+        [Range(1, 2)]
+        public int Choice { get; set; }
```
Comment (robinoconnor) [inline src/SFA.DAS.EmployerApprenticeshipsService.Application/Queries/GetSentTransferConnectionInvitationQuery/SentTransferConnectionInvitationViewModel.cs]: Is there a way to make choice more descriptive as other a developer has to trawl through the code to find out what each choice means?

File: src/SFA.DAS.EAS.Web/Controllers/TransferConnectionInvitationController.cs [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        public async Task<ActionResult> Create(CreateTransferConnectionInvitationViewModel model)
+        {
+            var transferConnectionInvitationId = await _mediator.SendAsync(model.Message);
+
+            return RedirectToAction("Send", new GetCreatedTransferConnectionInvitationQuery { TransferConnectionInvitationId = transferConnectionInvitationId });
+        }
+
+        [HttpNotFoundForNullModel]
+        [Route("{transferConnectionInvitationId}/send")]
+        public async Task<ActionResult> Send(GetCreatedTransferConnectionInvitationQuery query)
```
[the remark this answers is not in the document]
Comment (elliotritchie) [reply src/SFA.DAS.EAS.Web/Controllers/TransferConnectionInvitationController.cs]: It's a get request and shouldn't need one.

File: src/SFA.DAS.EAS.Web/Controllers/TransferConnectionInvitationController.cs [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+                    return RedirectToAction("Sent", new GetSentTransferConnectionInvitationQuery { TransferConnectionInvitationId = model.Message.TransferConnectionInvitationId });
+                default:
+                    return RedirectToAction("Index", "Transfer");
+            }
+        }
+
+        [HttpNotFoundForNullModel]
+        [Route("{transferConnectionInvitationId}/sent")]
+        public async Task<ActionResult> Sent(GetSentTransferConnectionInvitationQuery query)
+        {
```
[the remark this answers is not in the document]
Comment (elliotritchie) [reply src/SFA.DAS.EAS.Web/Controllers/TransferConnectionInvitationController.cs]: It's a get request and shouldn't need one.

File: src/SFA.DAS.EAS.Web/Controllers/TransferConnectionInvitationController.cs [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+            var response = await _mediator.SendAsync(query);
+            var model = _mapper.Map<SentTransferConnectionInvitationViewModel>(response);
+
+            return View(model);
+        }
+
+        [HttpPost]
+        [Route("{transferConnectionInvitationId}/sent")]
+        public ActionResult Sent(SentTransferConnectionInvitationViewModel model)
```
[the remark this answers is not in the document]
Comment (elliotritchie) [reply src/SFA.DAS.EAS.Web/Controllers/TransferConnectionInvitationController.cs]: There's no data modification but I've added one just in case, thanks :)

**Pull request merged**